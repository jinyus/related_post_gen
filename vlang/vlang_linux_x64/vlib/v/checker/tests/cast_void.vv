num := int(print(''))
println(num)
