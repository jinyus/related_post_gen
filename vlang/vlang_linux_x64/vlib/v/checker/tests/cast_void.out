vlib/v/checker/tests/cast_void.vv:1:12: error: expression does not return a value so it cannot be cast
    1 | num := int(print(''))
      |            ~~~~~~~~~
    2 | println(num)
